OasisDEX/oasis-borrow
# collateralisation ratio Fix

# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
![image](https://user-images.githubusercontent.com/10146412/132852663-67216eb9-f002-4004-b8d6-3e46df9eb604.png)

1)Open Vault
2) Go to Adjust Multiply Screen
3) move slider close to Collateralisation Ratio
4) Observe that there is no error
    
## Definition of done ✔️
